app/stacks/uk-west/applications-service/terragrunt.hcl

```diff
@@ -59,6 +60,7 @@ inputs = {
   back_office_service_bus_nsip_project_update_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_project_update_topic_id
   back_office_service_bus_nsip_representation_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_representation_id
   back_office_service_bus_nsip_exam_timetable_topic_id          = dependency.back_office_ukw.outputs.service_bus_nsip_exam_timetable_id
+  back_office_service_bus_nsip_advice_topic_id                  = dependency.back_office_ukw.outputs.service_bus_nsip_advice_topic_id
```

Comment:
We're going to need Back Office to add the `nsip-s51-advice` topic ID to the `back-office` stack's outputs (or add it as part of this PR). I'm not seeing it in `app/stacks/uk-west/back-office/outputs.tf`

Reply:
Think I have updated it